Diagnostics/HealthChecker/Helpers/Invoke-SetOutputInstanceLocation.ps1

```diff
@@ -0,0 +1,23 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Invoke-SetOutputInstanceLocation {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$FileName,
+
+        [Parameter(Mandatory = $false)]
+        [string]$Server,
+
+        [Parameter(Mandatory = $false)]
+        [bool]$IncludeServerName = $false
+    )
+    $endName = "-{0}.txt" -f $Script:dateTimeStringFormat
+
+    if ($IncludeServerName) {
+        $endName = "-{0}{1}" -f $Server, $endName
+    }
+
+    $Script:OutputFullPath = "{0}\{1}{2}" -f $Script:OutputFilePath, $FileName, $endName
```

Comment:
I think this one should also work
```suggestion
    $Script:OutputFullPath = [System.IO.Path]::Combine($Script:OutputFilePath, $FileName, $endName)
```

Reply:
This actually wouldn't work as you have it.

![image](https://user-images.githubusercontent.com/22776718/206760607-7e35c272-55e1-4894-a17a-09bbd9f49f40.png)

But could do something like this: 

![image](https://user-images.githubusercontent.com/22776718/206760991-c71822e2-a5c1-423a-b6c4-b2e03a355b8a.png)

Reply:
You're right. Let's use your suggestion or keep it as is.